tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -36,7 +36,7 @@ PACKAGE_LIST="autotools-dev \
         language-pack-en \
         libopenblas-dev \
         liblttng-ust0 \
-        libcurl3 \
+        libcurl4 \
```

Comment:
i thought we discussed making install_ubuntu.sh support both ubuntu 16.04 and 18.04
via an option.

Reply:
libcurl3 is for .net core

Reply:
i see. according to https://askubuntu.com/questions/1058125/curl-libcurl3-held-back-possibly-by-net-core , a newer version of dotnet-runtime can support libcurl4

Reply:
more info: https://github.com/dotnet/core-setup/issues/3875

Reply:
Can we just install both of them?

Reply:
> Can we just install both of them?

reading through the 2nd link, seems like we can't. anyway, this is just for TRT CI, we can make TRT work on ubuntu18, and keep others using ubuntu16